### PR TITLE
quality checks api

### PR DESCRIPTION
**Pull request by youngyjd**
### Summary of Changes

_Include a summary of changes then remove this line_

### Tests

_What tests did you add or modify and why? If no tests were added or modified, explain why. Remove this line_

### Documentation

_What documentation did you add or modify and why? Add any relevant links then remove this line_

### CheckList

Make sure you have checked **all** steps below to ensure a timely review.

- [ ] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [ ] PR includes a summary of changes.
- [ ] PR adds unit tests, updates existing unit tests, **OR** documents why no test additions or modifications are needed.
- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain docstrings that explain what it does
- [ ] PR passes `make test`

### CODE REVIEW

**Comment by dorianj** [comment]
Is there an RFC or other context for how this fits into bigger picture (what other components there are if any, how this is used, etc)? This is neat, and I can surmise, but would love to review this with that bigger-picture context. not saying it necessarily needs an rfc, but would good to have some info about the scope

File: metadata_service/api/data_quality_checks.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+from metadata_service.proxy import get_proxy_client
+
+
+class TableQualityChecksAPI(Resource):
+    def __init__(self) -> None:
+        self.client = get_proxy_client()
+        super(TableQualityChecksAPI, self).__init__()
+
+    @swag_from('swagger_doc/data_quality_checks/table_quality_checks_get.yml')
+    def get(self, table_uri: str) -> Tuple[Any, HTTPStatus]:
```
Comment (mgorsk1) [inline metadata_service/api/data_quality_checks.py:20]: would make sense to provide (in time) put/post api so external systems can dynamically register new checks

File: metadata_service/proxy/neo4j_proxy.py [hunk 2]
```diff
@@ -1412,3 +1413,6 @@ def get_lineage(self, *,
                     id: str,
                     resource_type: ResourceType, direction: str, depth: int) -> Lineage:
         pass
+
+    def get_table_quality_checks(self, *, table_uri: str) -> List[TableQualityCheck]:
```
Comment (mgorsk1) [inline metadata_service/proxy/neo4j_proxy.py:1417]: needs to also be implemented in atlas_proxy otherwise it breaks it

**Review by mgorsk1 — COMMENTED** [review]
looks promising, agree with @dorianj some more context on this would be nice

**Comment by youngyjd** [comment]
Thanks everyone for the comment. After talking with the team, we have decided to make this table quality check api internally first. Once it work out well and steady on our side, I will make a RFC and see if people are interested in it.

**Comment by dorianj** [comment]
I saw in the other thread this was going to be upstreamed after it was tested internally, so closing. Looking forward to it, though!

**Pull request closed without merging**